contracts/contract_state_tree.md

```diff
@@ -62,3 +62,7 @@ the latest block only contains the calls in that block.
 The key is `<contract_address><caller_address><caller_nonce><contract_address>`.
 
 See [Call Serialization](../serializations.md#contract-call) for the serialization specification.
+
+Each call includes the caller address and caller nonce that, together,
+correlate the call with the transaction in the block that did the
```

Comment:
I see the code implements [correlation from contract call tx (of known hash) to contract call object](https://github.com/aeternity/epoch/blob/3e5e28a1452e8b7e17af9f940381c72b8d690d4d/apps/aehttp/src/aehttp_helpers.erl#L182-L189) (implemented by https://github.com/aeternity/epoch/pull/1185 ).

Not sure about correlation - if any - from contract create tx to contract call object.
Not sure about correlation - if any - from contract created from contract call (if any) to contract call object.

Cc @happi